docs/general-settings.md

```diff
@@ -20,5 +20,10 @@ settings: {
 * **header.logo** defines the top left navigation logo. It has a fixed height of 28px.
 * **header.title** defines the top left navigation title.
 * **header.favicon** defines the favicon. It rquires a standard favicon file with the `.ico` extension, and 16x16px or 32x32px dimensions.
-* **responsiveNavigation** allows adding a button on the left side of the top navigation. Upon click, the button shows or hides the left navigation. The possible values are `simple` and `simpleMobileOnly`. `simple` displays the button regardless of the browser window´s size, while `simpleMobileOnly` shows the button when the width is lower than `600px`. If you don't specify any value for  **responsiveNavigation**, the button will not be displayed. The same applies when you enable **hideSideNav** for the current active navigation node. 
-* **sideNavFooterText** is a string displayed in a sticky footer inside the Side Navigation. It is a good place to display the version of your application.
+* **responsiveNavigation** allows customizing the navigation display settings. For example, you can define buttons which show or completely hide the left navigation, or collapse it to only show the icons. 
+You can set the following values:
+* `simple` displays the button regardless of the browser window´s size.
```

Comment:
For the first two options, the information about where is the button displayed and what happens when we click it is missing (before we had _on the left side of the top navigation. Upon click, the button shows or hides the left navigation_). Please check the content we had before ;)

Reply:
It's a valid point - I thought that  I may simplify it a little bit and provide the descriptions of buttons as an example, but you are right that we need to be more exact. The question is: this part `if you don't specify any value for responsiveNavigation, the button will not be displayed.` - does it apply only to the button on the left side of the top navigation or also to the one that is at the bottom and collapses the navigation?